package.json

```diff
@@ -73,6 +73,7 @@
     "karma-sourcemap-loader": "0.3.7",
     "karma-webpack": "2.0.3",
     "loader-utils": "1.1.0",
+    "lodash.merge": "^4.6.0",
```

Comment:
For consistency with how we're consuming other libraries, let's make the version exact.